packages/node_modules/@ciscospark/plugin-people/src/people-batcher.js

```diff
@@ -18,8 +18,11 @@ const PersonUUIDRequestBatcher = Batcher.extend({
    * @returns {Promise}
    */
   handleHttpSuccess(res) {
+    // combine item and notFoundIds lists
+    const notFounds = res.body.notFoundIds || [];
     return Promise.all(res.body.items.map((personResponse) =>
-      this.handleItemSuccess(personResponse.id, personResponse)));
+      this.handleItemSuccess(personResponse.id, personResponse))
+      .concat(notFounds.map((id) => this.handleItemSuccess(id))));
```

Comment:
If we are going through the work to remove these in `people.js`, what benefit do we get from `.concat`ing them?

Reply:
Way to get to the heart of it Adam. In truth I wasn't sure what batch needed underneath to know when a request had finished. Now after looking through underlying code, what matters is all the request made in people.js line 50 resolve. The return value from handleHTTPSuccess doesn't matter at all for our purposes. 

So a misunderstanding, good catch Adam. I'll move the notFoundIds handling to a resolver after PeopleBatcher line 23 Promise.all